chia_py_streamable_macro/src/lib.rs

```diff
@@ -71,6 +71,12 @@ pub fn py_streamable_macro(input: TokenStream) -> TokenStream {
                     pub fn from_json_dict(o: &pyo3::PyAny) -> PyResult<Self> {
                         <Self as FromJsonDict>::from_json_dict(o)
                     }
+
+                    #(#[getter]
+                        fn #fnames(&self, py: Python) -> PyResult<#ftypes> {
+                            Ok(self.#fnames.clone())
+                        }
+                    )*
```

Comment:
this is the part of the macro that creates the getter functions, replacing the need for `#[pyo3(get)]`

Reply:
Nice. It looks like this creates getter functions for all members. You might want to add a note about expected use of mutable members if you are keeping an API guidelines doc.

Reply:
all these types we export (so far) are use the same way our `dataclass(frozen=True` are in python, i.e. immutable.
I don't expect we'll have a type where python communicates back to rust by mutating members either.